src/System.Private.CoreLib/shared/Internal/Runtime/CompilerServices/Unsafe.cs

```diff
@@ -387,5 +387,45 @@ public static IntPtr ByteOffset<T>(ref T origin, ref T target)
         {
             throw new PlatformNotSupportedException();
         }
+
+        /// <summary>
+        /// Returns a by-ref to a null reference of type <typeparamref name="T"/>.
+        /// Note this is the address of the by-ref not what the ref points to.
```

Comment:
These two lines sounds contradicting to me. Should it rather read "Returns a by-ref that is a null reference of type..."?

Reply:
"Returns a by-ref that is a null reference **to** a type of T." ?